config/default/testRunner.conf.php

```diff
@@ -388,6 +388,10 @@
         ],
         'previous' => [
             'trigger' => 'K'
+        ],
+        'dialog' => [
+            'accept' => 'Enter',
+            'reject' => 'X'
```

Comment:
Ok, so let keep the `<esc>` key as a default. I will provide a dedicated config for the customer who need to deal with fullscreen.